code/modules/clothing/suits/armor.dm

```diff
@@ -55,14 +55,52 @@
 	slowdown = LIGHT_SLOWDOWN
 	style_coverage = COVERS_TORSO|COVERS_UPPER_ARMS|COVERS_UPPER_LEGS
 
-/obj/item/clothing/suit/armor/vest/full/security
+
+//This has specifically been bodged so that I can give the flak vest toggle-able buttons -VaNdU Jr
+/obj/item/clothing/suit/armor/vest/toggleable/security/full
```

Comment:
For consistency sake all instances of `toggleable` should be `toggle` instead (toggleable is used for items/devices and toggle for clothes)

Reply:
can do

Reply:
Fixed it, resolving
if it's not to your liking, please unresolve.